public/javascripts/items.js

```diff
@@ -10,25 +10,83 @@ const fetchParams = {
 const params = completed =>
   Object.assign( {}, fetchParams, { body: JSON.stringify({ completed }) })
 
+const params2 = title =>
+  Object.assign( {}, fetchParams, { body: JSON.stringify({ title }) })
+
+const params3 = description =>
+  Object.assign( {}, fetchParams, { body: JSON.stringify({ description }) })
+
 $(document).ready( () => {
-  $( '.title' ).click( event => {
+  $('.updateTitle').keypress( event => {
+    element = $( event.target )
+    const id = element.data( 'id' )
+
+    if( event.charCode === 13 ) {
+      let updatedTitle = element[0].value
+      fetch( `/items/${id}/edit`, params2( updatedTitle ))
+      .then( result => result.json() )
+      .then( json => {
+        if( json.success ) {
+          let span = element.prev()
+          $( span[0] ).html(updatedTitle)
+          element.addClass( 'hidden')
+          $( span ).removeClass( 'hidden' )
+        }
+      })
+    }
+  })
+
+  $( '.title > span' ).click( event => {
     const element = $( event.target )
+    $( element[0] ).addClass( 'hidden' )
+    let input = element.next()
```

Comment:
Same as above - can we look for a means of more explicitly identifying the elements we will want to update?